tests/models/test_model.py

```diff
@@ -17,9 +19,37 @@
 
 from unittest import mock
 from scipy.sparse import csc_matrix
+import sklearn.datasets
+import sklearn.neighbors
 from packaging.version import Version
 
 
+def _load_pyfunc(path):
+    with open(path, "rb") as f:
+        return pickle.load(f, encoding="latin1")  # pylint: disable=unexpected-keyword-arg
```

Comment:
Is this function used somewhere?

Reply:
Is this for loading a model in `test_validate_schema`?

Reply:
Yes.

Reply:
Do we need to specify `encoding="latin1"`?

Reply:
You're right, this is not needed